source/renderer/app/stores/WalletsStore.js

```diff
@@ -142,7 +142,9 @@ export default class WalletsStore extends Store {
   /* ----------  Active Wallet  ---------- */
   @observable active: ?Wallet = null;
   @observable activeValue: ?BigNumber = null;
-  @observable activePublicKey: ?string = null;
+  @observable walletsPublicKeys: {
```

Comment:
@aleksandardjordjeviciohk this needs to be reverted. We only keep **one** public key in the store.

Reply:
Reverted